modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/UserManager.java

```diff
@@ -46,8 +46,25 @@ public User getUser(String id, Map<String, Boolean> requiredAttributes)
     public void deleteUser(String userId)
             throws NotFoundException, CharonException, NotImplementedException, BadRequestException;
 
+    default List<Object> listUsersWithGET(Node node, Integer startIndex, Integer count, String sortBy, String sortOrder,
+            String domainName, Map<String, Boolean> requiredAttributes)
+            throws CharonException, NotImplementedException, BadRequestException {
```

Comment:
add a new line after method declaration

Reply:
Fixed in https://github.com/wso2/charon/pull/263